lib/mixins/property-accessors.html

```diff
@@ -436,12 +463,41 @@
        * @protected
        */
       _flushProperties() {
-        let oldProps = this.__dataOld;
-        let changedProps = this.__dataPending;
-        this.__dataPending = null;
-        this.__dataCounter++;
-        this._propertiesChanged(this.__data, changedProps, oldProps);
-        this.__dataCounter--;
+        if (!this.__dataInitialized) {
```

Comment:
Now that we have `__dataInitialized` it probably makes sense to prevent `_invalidateProperties` from scheduling a flush if this flag is not true.